kubernetes-core.el

```diff
@@ -46,21 +46,21 @@
           (magit-section-update-highlight))))))
 
 
-(defmacro kubernetes--save-window-state (&rest body)
+(defun kubernetes--save-window-state (&rest body)
   "Restore window state after executing BODY.
 
 This is useful if the buffer is erased and repopulated in BODY,
 in which case `save-excursion' is insufficient to restore the
 window state."
-  `(let ((pos (point))
-         (col (current-column))
-         (window-start-line (window-start))
-         (inhibit-redisplay t))
-     (save-excursion
-       ,@body)
-     (goto-char pos)
-     (move-to-column col)
-     (set-window-start (selected-window) window-start-line)))
+  (let ((pos (point))
+        (col (current-column))
+        (window-start-line (window-start))
+        (inhibit-redisplay t))
+    (save-excursion
+       ',@body)
```

Comment:
Will "expansion" of `body` expressions continue to work if we redefine this as a function rather than a macro?

Reply:
It should yield `(1)` but it won't since I used single-quote instead of back tick ...